bin/index.js

```diff
@@ -49,7 +51,7 @@ try {
   main = require(path.join(__dirname, '..', 'commands', `${command}.js`))
 } catch (err) {
   console.error(chalk.red(`
-Unknown command: ${command}`))
+Unknown command: ${command}`, err))
```

Comment:
Yeah, does someone wanting `cordova build --debug` also want to see stack traces and other internal data from the CLI tool? Probably not. Alternatively, we always do both debug and production builds, giving users both APKs. Then it frees up the flag for other purposes. /shrug